phpBB/includes/functions_user.php

```diff
@@ -2124,7 +2124,8 @@ function group_correct_avatar($group_id, $old_entry)
 
 	try
 	{
-		$storage->rename($old_filename, $new_filename);
+		$storage->write($new_filename, $storage->read($old_filename));
```

Comment:
I actually like the old rename, any reason to remove it?

Reply:
Yes, because is not used anywhere, only here, and this is probably never executed.